advising/urls.py

```diff
@@ -36,10 +36,10 @@
     # Get the courses for the requesting user
     url(r'^advising/sis_courses/(?P<sem_name>.+)/(?P<year>[0-9]{4})/$',
         advising.views.RegisteredCoursesView.as_view()),
-    # Get the courses for the student with this JHED
-    url(r'^advising/sis_courses/(?P<sem_name>.+)/(?P<year>[0-9]{4})/(?P<jhed>.+)/$',
+    # Get the courses for the student with this JHED (Empty value for tt_name must be passed)
+    url(r'^advising/sis_courses/(?P<sem_name>.+)/(?P<year>[0-9]{4})//(?P<jhed>.+)/$',
```

Comment:
Looks like you added in an extra '/' here?

Reply:
that's the only way the two URLs work. If not, it always assumes that the URL without the tt_name is being used. So, if i ask for the view to print out the JHED - it gives me <timetable_name>/<jhed>. Adding the extra / means that if one the frontend, nothing is passed in for ttname, the URL it is calling is advising/sis_courses/<sem_name>/<year>//<jhed>/' since tt_name would be empty. It is also a solution to scrap that URL altogether and rely on the one below it - but this was my fix for now.